packages/js/src/configs/eslint-recommended.js

```diff
@@ -9,7 +9,7 @@
 /* eslint sort-keys: ["error", "asc"] -- Long, so make more readable */
 
 /** @type {import("../lib/shared/types").ConfigData} */
-module.exports = {
+module.exports = Object.freeze({
```

Comment:
Hint: [`Object.freeze`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze#what_is_shallow_freeze) just protects the **first** obj _layer_. Nested objects can be manipulated like this:
```js
let o = Object.freeze({rules: {a: 1, b: 2}});
o.rules.a = 'overwrite';
delete o.rules.b; // delete

console.log(JSON.stringify(o)); // {"rules":{"a":"overwrite"}}
```
A more failsafe solution might be recursive `deep_freeze(obj)` fn, since rules might also be defined as objects. 🤔

Reply:
Yup, I'm aware. :) Forgot to go back and add this.